doc/user/content/sql/create-sink.md

```diff
@@ -148,171 +129,19 @@ Note that:
-{consistency_topic_prefix}-{sink_global_id}-{materialize-startup-time}-{nonce}
-```
-
-{{< note >}}
-If `reuse_topic` is enabled, the naming convention is ignored. Instead, the topic name specified via the `CONSISTENCY` option is used.
-{{</ note >}}
-
-Each message in the consistency topic has the schema below.
-
-```json
-{
-    "type": "record",
-    "name": "envelope",
-    "fields": [
-        {
-            "name": "id",
-            "type": "string"
-        },
-        {
-            "name": "status",
-            "type": "string"
-        },
-        {
-            "name": "event_count",
-            "type": [
-                null,
-                "long"
-            ]
-        },
-        {
-            "name": "data_collections",
-            "type": [
-                "null",
-                {
-                    "type": "array",
-                    "items": {
-                        "type": "record",
-                        "name": "data_collection",
-                        "fields": [
-                            {
-                                "name": "data_collection",
-                                "type": "string"
-                            },
-                            {
-                                "name": "event_count",
-                                "type": "long"
-                            },
-                        ]
-                    }
-                }
-            ],
-            "default": null,
-        }
-    ]
-}
-```
-
-Field | Use
-------|-----
-_id_ | The transaction `id` this record refers to.
-_status_ | Either `BEGIN` or `END`. Materialize sends a record with `BEGIN` the first time it writes a data message for `id`, and an `END` record after it has written all data messages for `id`.
-_event&lowbar;count_ | This field is null for `BEGIN` records, and for `END` records it contains the number of messages Materialize wrote for that `id`.
-_data&lowbar;collections_ | This field is null for `BEGIN` records, and for `END` records it contains the number of messages Materialize wrote for that `id` and collection.
-
-##### Consistency information details
-- Materialize writes consistency output to a different topic per sink.
-- There are no ordering guarantees on transaction `id` in the consistency topic.
-- Multiple transactions can be interleaved in the consistency topic, so it's possible that some `ids` don't have a corresponding `BEGIN` or `END` record.
+To achieve its exactly-once processing guarantees, Materialize needs to store some internal metadata in an additional *progress topic*. This topic is shared among all sinks that use a particular Kafka connection. The name of this progress topic can be specified when [creating a connection](/sql/create-connection); otherwise, a default is chosen based on the Materialize environment id and the connection id. In either case, Materialize will attempt to create the topic if it does not exist. The contents of this topic are not specified.
```

Comment:
```suggestion
To achieve its exactly-once processing guarantees, Materialize needs to store some internal metadata in an additional *progress topic*. This topic is shared among all sinks that use a particular Kafka connection. The name of this progress topic can be specified when [creating a connection](/sql/create-connection); otherwise, a default is chosen based on the Materialize environment `id` and the connection `id`. In either case, Materialize will attempt to create the topic if it does not exist. The contents of this topic are not user-specified.
```

Reply:
> The contents of this topic are not specified.

Does "user-specified" sound reasonable? It's a bit harder to understand the meaning of the sentence otherwise!